### PR TITLE
Added build script for oas-kit

### PR DESCRIPTION
**Pull request by atharv-phadnis**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?

Package Name: **oas-kit**
Package Version: **v5.0.0**
Package URL: [oas-kit](https://github.com/Mermade/oas-kit)

Tested on: **UBI 8.5**
Build Errors: **None**
Test Errors: **None**

### CODE REVIEW

**Comment by atharv-phadnis** [comment]
@Siddhesh-Ghadi please review

File: o/oas-kit/oas-kit.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=oas-kit
+PACKAGE_VERSION=v5.0.0
```
Comment (seth-priya) [inline o/oas-kit/oas-kit.sh:21]: can we make this configurable - allow user to pass it as an argument to the script?

File: o/oas-kit/oas-kit.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: oas-kit
+# Version	: v5.0.0
+# Source repo	: https://github.com/Mermade/oas-kit
+# Tested on	: UBI 8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: BulkPackageSearch Automation {maintainer}
```
Comment (seth-priya) [inline o/oas-kit/oas-kit.sh:10]: this should be updated to you name + email?

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: o/oas-kit/oas-kit.sh (hunk 2 above)
Comment (atharv-phadnis) [reply o/oas-kit/oas-kit.sh]: Updated, please review

File: o/oas-kit/oas-kit.sh (hunk 1 above)
Comment (atharv-phadnis) [reply o/oas-kit/oas-kit.sh]: Updated

**Review by seth-priya — APPROVED** [review]

**Pull request merged**